packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -109,6 +110,22 @@ export const processPatientDocumentRequest = async (
       );
     }
 
+    if (
+      !Config.isProdEnv() &&
```

Comment:
not happening in prod

Reply:
The whole idea of events is to always fire them and not be specific to any listener/subscriber.

We should:
- fire a generic event: `patientEvents().emitDocumentWebhookSent()`
   - including `webhookRequest`, `patientId`
- on `canvas-event-listener.ts` we can add the logic whether to execute the logic or not based on the event's data.

This approach makes the current file (emitter) to be agnostic of who's listening and their logic.